docs/topolvm-controller.md

```diff
@@ -158,7 +158,7 @@ Controllers
 
 `topolvm-metrics` adds `topolvm.cybozu.com/node` finalizer.
 
-When a Node is being deleted, the controller deletes all PVCs for TopoLVM
+When a Node is being deleted, the controller deletes all PVCs and Logical Volumes for TopoLVM
```

Comment:
Space confuses me (but the naming is actually confusing...)
Would you use `LogicalVolumes`?

Reply:
fixed it.